polyfills/Intl/RelativeTimeFormat/tests.js

```diff
@@ -2859,9 +2859,9 @@ describe("Intl.RelativeTimeFormat", function () {
     proclaim.equal(
       new Intl.RelativeTimeFormat("en", { style: "narrow" }).format(
         3,
-        "quarter"
+        "month"
       ),
-      "in 3 qtr"
+      "in 3 mo."
```

Comment:
@longlho should this be:
```suggestion
      "in 3 mo"
```

The current version fails in IE 9, 10 and 11.

```
Failures:
 - ie/10.0:
    -> Intl.RelativeTimeFormat Intl.RelativeTimeFormat format should work for style narrow
       http://bs-local.com:9876/test?includePolyfills=yes&always=no&feature=Intl.RelativeTimeFormat
       in 3 mo == in 3 mo.
 - ie/11.0:
    -> Intl.RelativeTimeFormat Intl.RelativeTimeFormat format should work for style narrow
Error: Failures detected
    at /home/runner/work/polyfill-library/polyfill-library/test/polyfills/remotetest.js:318:13
       http://bs-local.com:9876/test?includePolyfills=yes&always=no&feature=Intl.RelativeTimeFormat
       in 3 mo == in 3 mo.
 - ie/9.0:
    -> Intl.RelativeTimeFormat Intl.RelativeTimeFormat format should work for style narrow
       http://bs-local.com:9876/test?includePolyfills=yes&always=no&feature=Intl.RelativeTimeFormat
       in 3 mo == in 3 mo.
```

Reply:
I pushed a fix. This is bc the CLDR data shipped with the polyfill is newer than Chrome's version (in the new version all `.` were removed so `mo.` -> `mo`, `qtr.` -> `qtr`). IE got the polyfill so it outputs `mo` but Chrome doesn't so it outputs `mo.`